warehouse/core/src/main/java/datawave/ingest/data/config/ingest/AccumuloHelper.java

```
@@ -30,10 +32,17 @@ public class AccumuloHelper {
     private String zooKeepers = null;
     private String accumuloClientPropertiesPath = null;
 
+    private static final Logger log = LoggerFactory.getLogger(AccumuloHelper.class);
+
     public void setup(Configuration config) throws IllegalArgumentException {
         username = ConfigurationHelper.isNull(config, USERNAME, String.class);
-        byte[] pw = Base64.decodeBase64(ConfigurationHelper.isNull(config, PASSWORD, String.class).getBytes());
-        password = new PasswordToken(pw);
+        String encodedPW = ConfigurationHelper.isNull(config, PASSWORD, String.class);
+        if (encodedPW.length() >= 2) {
+            password = new PasswordToken(Base64.getDecoder().decode(encodedPW));
+        } else {
+            log.warn("no Accumulo password?");
```

only a warning?

---

I don't think the check on `encodedPW.length()` is needed here. Just let the exception be thrown if the ConfigurationHelper returns garbage.

---

changed